static/app/views/releases/thresholdsList/index.tsx

```diff
@@ -1,36 +1,180 @@
-import {useEffect} from 'react';
-import {RouteComponentProps} from 'react-router';
+import {useEffect, useMemo} from 'react';
+import styled from '@emotion/styled';
 
-// import {Organization, PageFilters, Project} from 'sentry/types';
+import GuideAnchor from 'sentry/components/assistant/guideAnchor';
+import EnvironmentPageFilter from 'sentry/components/environmentPageFilter';
+import * as Layout from 'sentry/components/layouts/thirds';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import NoProjectMessage from 'sentry/components/noProjectMessage';
+import PageFilterBar from 'sentry/components/organizations/pageFilterBar';
+import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
+import PanelTable from 'sentry/components/panels/panelTable';
+import ProjectPageFilter from 'sentry/components/projectPageFilter';
+// import {ALL_ACCESS_PROJECTS} from 'sentry/constants/pageFilters';
+import {t} from 'sentry/locale';
+// import ConfigStore from 'sentry/stores/configStore';
+import {space} from 'sentry/styles/space';
+// import {Project} from 'sentry/types';
 import useOrganization from 'sentry/utils/useOrganization';
-
-// import usePageFilters from 'sentry/utils/usePageFilters';
+import usePageFilters from 'sentry/utils/usePageFilters';
 // import useProjects from 'sentry/utils/useProjects';
-// import useRouter from 'sentry/utils/useRouter';
+import useRouter from 'sentry/utils/useRouter';
+
 import Header from '../components/header';
+import {Threshold} from '../utils/types';
+import useFetchThresholdsListData from '../utils/useFetchThresholdsListData';
 
-type RouteParams = {
-  orgId: string;
-};
+import {ThresholdGroupRow} from './thresholdGroupRow';
 
-type Props = RouteComponentProps<RouteParams, {}> & {};
+type Props = {};
 
-function ReleaseThresholdList({router}: Props) {
+function ReleaseThresholdList({}: Props) {
+  const router = useRouter();
   const organization = useOrganization();
   useEffect(() => {
     const hasV2ReleaseUIEnabled = organization.features.includes('release-ui-v2');
     if (!hasV2ReleaseUIEnabled) {
       router.replace('/releases/');
     }
   }, [router, organization]);
-  // const {projects, initiallyLoaded: projectsLoaded} = useProjects();
-  // const {selection, isReady, desyncedFilters} = usePageFilters();
+  // const {projects} = useProjects();
+  const {selection} = usePageFilters();
+  const {
+    data: thresholds = [],
+    error,
+    isLoading,
+    isError,
+    refetch,
+  } = useFetchThresholdsListData({
+    selectedProjectIds: selection.projects,
+  });
+
+  // const _getAllSelectedProjects = (): Project[] => {
```

Comment:
If the comments are not fast follow-up remove them to avoid confusion.

Reply:
Yea - these are fast follow up items
Will be utilized once this page is complete